RegisterScreen, Index, settings  Überarbeitet

Dstarke28 opened the pull request:
- RegisterScreen complete with Database Connection
- Index.js navigator Fehler behoben
- conn settings default "richtig überarbeitet"

Commit:
- RegisterScreen complete with Database Connection
- Index.js navigator Fehler behoben
- conn settings default "richtig überarbeitet"

rubengees commented on screens/RegisterScreen.js:
```diff
@@ -163,8 +99,71 @@ class RegisterScreen extends Component {
-        Alert.alert('Sie wurden mit folgenden Angaben:', pFName +"\n"+ pLName +"\n"+ pUsername +"\n"+ pPasswort +"\n"+ pAge +"\n"+ pEyecolor +"\n"+ pHaircolor +"\n"+ pSex +"\n"+ pBodyheight +"\n"+ pFigure, [{text: 'erfolgreich registriert'}])
+    _createUser(pUsername, pPassword, pPasswordTest, pCallback ) {
+
+        var fehler = false;
```
-.-

rubengees commented on components/backend/conn-settings.js.default:
```diff
@@ -1,5 +1,7 @@
 var connSettings = {
-    url: "",
+    protocol: "",
+    url: "e",
```
Was hat das "e" zu bedeuten? Typo?

rubengees commented on screens/RegisterScreen.js:
```diff
@@ -163,8 +99,71 @@ class RegisterScreen extends Component {
+            Alert.alert('Fehler', "Password nicht Identisch" , [{text: 'ok'}]);
+            fehler = true;
+        }
+        else if ( fehler == false ) {
```
Das ist unnötig. Diese Abzweigung würde so oder so nur aufgerufen werden, wenn keine der obigen Abfragen wahr ist.

rubengees commented:
Beim manuellen Testen, werde ich bei einer Fehlangabe zurück auf den Loginscreen verwiesen. Bitte einmal überprüfen wie das kommt.

Commit:
- RegisterScreen v2.0
- Das 'e' in der url hat nichts zu bedeuten Tippfehler, Sry
- Das wir weiter geleitet werden, ist noch ein Fehler im callback, da kann ich derzeit nichts dran ändern.

Commit:
- Login Screen fixed
- Register Screen  navigateTo fixed
